README.md

```diff
@@ -1,2 +1,219 @@
+
+If everything has been set up properly, after completing the workflow approximately the following structure will be created on the server.
+Check how the on your server folder structure would look like after one, two and three deployments.
+
+```
+-- /var/www/my-app.com
+|-- current -> /var/www/my-app.com/releases/20100509145325
+|-- releases
+|   |-- 20100509145325
+|-- shared
+```
+
+```
+-- /var/www/my-app.com
+|-- current -> /var/www/my-app.com/releases/20100509150741
+|-- releases
+|   |-- 20100509150741
+|   |-- 20100509145325
+|-- shared
+```
+
+```
+-- /var/www/my-app.com
+|-- current -> /var/www/my-app.com/releases/20100512131539
+|-- releases
+|   |-- 20100512131539
+|   |-- 20100509150741
+|   |-- 20100509145325
+|-- shared
+```
+
+### Rolling back
+
+In order to rollback, you need to set up the deployment and run the rollback workflow. Workflow, will switch the current folder to the previous release.
+- Create a new workflow file for rollback, for example, [`rollback.yml`](https://github.com/saritasa-nest/ceai-wordpress/blob/develop/.github/workflows/ci-rollback.yml)
+- Place the created playbook in your repository for example, [`ci\ansible\rollback.yaml`](https://github.com/saritasa-nest/ceai-wordpress/blob/develop/ci/ansible/ansistrano-deploy.yml)
+
+Watch the [video](https://vimeo.com/manage/videos/869032751)
+
+Before rollback
+```
+-- /var/www/my-app.com
+|-- current -> /var/www/my-app.com/releases/20100512131539
+|-- releases
+|   |-- 20100512131539
+|   |-- 20100509150741
+|   |-- 20100509145325
+|-- shared
+```
+
+After rollback
+```
+-- /var/www/my-app.com
+|-- current -> /var/www/my-app.com/releases/20100509150741
+|-- releases
+|   |-- 20100509150741
+|   |-- 20100509145325
+|-- shared
+```
+
+### Inputs variables
+
+| Variable       | Type          | Example                  | Discription                                                                                                       |
+| ---------------|:-------------:| :------------------------|-------------------------------------------------------------------------------------------------------------------|
+| server_host    | string        | 59.128.16.24             | The address of the server for deploy                                                                              |
+| environment    | string        | production               | The environment for deploy i.e development, staging, production                                                   |
+| playbook_path  | string        | ci/ansible/playbook.yaml | The path to the file relative to the repository from which the deployment is made, used in Ansistrano deploy task |
+| deploy_path    | string        | ~/deploy/                | The path to deploy on the remote server                                                                           |
```

Comment:
is it important to use trailing "/" for this variable?

Reply:
The presence of the final "/" optional - fixed